go.mod

```diff
@@ -14,15 +14,15 @@ require (
 	github.com/containers/buildah v1.29.1-0.20230201192322-e56eb25575c7
 	github.com/containers/common v0.51.0
 	github.com/containers/conmon v2.0.20+incompatible
-	github.com/containers/image/v5 v5.24.1-0.20230202144111-a49c94a010be
+	github.com/containers/image/v5 v5.24.1-0.20230208161124-34839152eb48
 	github.com/containers/ocicrypt v1.1.7
 	github.com/containers/psgo v1.8.0
 	github.com/containers/storage v1.45.3
 	github.com/coreos/go-systemd/v22 v22.5.0
 	github.com/coreos/stream-metadata-go v0.0.0-20210225230131-70edb9eb47b3
 	github.com/cyphar/filepath-securejoin v0.2.3
 	github.com/digitalocean/go-qemu v0.0.0-20210326154740-ac9e0b687001
-	github.com/docker/docker v20.10.23+incompatible
```

Comment:
This will likely need to be pinned to the previous version to wait until imagebuilder is compatible with the new API

Reply:
This PR already includes an updated `openshift/imagebuilder`. (At least it seems to build and pass tests; it’s possible that it’s broken in some other known way I am unaware of.)

Reply:
Oh nice, thanks!